scripts/build-package

```diff
@@ -3,6 +3,8 @@ set -e
 
 pip install --require-hashes -r requirements.txt --target ./package
 pip install --no-deps --target ./package .
+git clone git+https://${GITHUB_TOKEN}@github.com/release-engineering/cdn-definitions-private.git
```

Comment:
Since I do not have the overall knowledge of this I am going to ask this question:
Will be the directory with cloned repo deleted before next run of this script? Right now when you run this multiple times it will not fetch latest changes, it just says directory already exists and is not empty.

Reply:
It looks like it might be run on AWS pipeline, and I guess it starts always with clear data storage? So you will not encounter that issue mentioned above. Am I right?

Reply:
> So you will not encounter that issue mentioned above. Am I right?

Yes, you are correct. This script is intended to be run by AWS CodeBuild. CodeBuild spins up a fresh build environment for each build.